engine/installation/linux/debian.md

```diff
@@ -146,12 +146,16 @@ from the repository.
     **armhf**:
 
     ```bash
-    $ sudo add-apt-repository \
-       "deb [arch=armhf] {{ download-url-base }} \
-       $(lsb_release -cs) \
-       stable"
+    $ echo "deb [arch=armhf] https://apt.dockerproject.org/repo \
+        raspbian-jessie main" | \
+        sudo tee /etc/apt/sources.list.d/docker.list
     ```
 
+    > **Warning**: It is HIGHLY recommended you use
+    > ```bash $ curl -sSL https://get.docker.com | sh ``` in order to install Docker on
+    > Raspberry Pi's!! This will also install bash auto-completion and
+    > appropriately mirror the download depending on your country.
+
```

Comment:
@NonCreature0714 just a minor comment to replace the double exclamations with just a period :-) We can leave the all-caps HIGHLY, but I think that and the warning formatting should be enough. (Giving the pros for doing this is great, but if you want to add any particular downside of _not_ using this command, that might be more compelling than the exclamation marks.)